src/attributes/classes.js

```diff
@@ -23,9 +39,7 @@ jQuery.fn.extend( {
 			} );
 		}
 
-		if ( typeof value === "string" && value ) {
-			classes = value.match( rnothtmlwhite ) || [];
```

Comment:
I've looked at the latest version. How many bytes does it add compared to the current master? You'll see that when you run `grunt`, the minified gzipped.